lib/cucumber-api/steps.rb

```diff
@@ -21,6 +21,16 @@
   }
 end
 
+Given(/^I set Headers to:$/) do |params|
+  @headers = {
```

Comment:
I prefer to remove this default `@headers`, and change this step def to `I add Headers:`, so you can use this in conjunction with current steps like `I send ? and accept JSON`